tests/bluetooth/tester/testcase.yaml

```diff
@@ -14,6 +20,13 @@ tests:
     extra_args: OVERLAY_CONFIG="overlay-le-audio.conf"
     tags: bluetooth
     harness: bluetooth
+  bluetooth.general.tester_le_audio_native:
+    build_only: true
+    platform_allow:
+      - native_posix
+    extra_args: OVERLAY_CONFIG="overlay-le-audio.conf;overlay-native.conf"
```

Comment:
What is the reason why we have a `overlay_native` rather than a `boards/native_posix.conf`? 

If we had the above, we should be able to just add `native_posix` to the `platform_allow` above, right?

Reply:
I've tried to do it exactly as you suggested, but west build does not apply the native_posix.conf for some reason.

Reply:
> I've tried to do it exactly as you suggested, but west build does not apply the native_posix.conf for some reason.

Hmm, strange. IIRC there is something about CI only picking and building for 1 of the possible targets, and not all (for each CI run). Maybe that's the case?

Reply:
I figured out what was the reason. The overlays and configurations were not located under ./board directory so that were not applied by build system